docs/modules/json.md

```diff
@@ -0,0 +1,71 @@
+# Json Module
+This module provides functionality from the [json widget](https://spectreconsole.net/widgets/json) of _Spectre.Console_.
+
+Json can be printed in the console using the the json function:"
```

Comment:
Too many "the"s (same is true in the original documentation command)

Reply:
That's that's true.